parlai/core/worlds.py

```diff
@@ -75,6 +76,63 @@ def __init__(self, opt, agents=None, shared=None):
             # Add passed in agents to world directly.
             self.agents = agents
 
+    def parley(self):
+        """ The main method, that does one step of actions for the agents
+        in the world. This is empty in the base class."""
+        pass
+
+    def getID(self):
+        """Return the name of the world, typically the task the world encodes."""
+        return self.id
+
+    def display(self):
+        """Returns a string describing the current state of the world.
+        Useful for monitoring and debugging.
+        By default, display the messages between the agents."""
+        lines = []
+        for index, msg in enumerate(self.acts):
```

Comment:
maybe self.get_acts(), with a `return []` base definition? this could get a runtime error right now since self.acts isn't defined in __init__.